fix bug in validate layout (#437)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

> * Fixed a bug in the **validate** command failed on "edit layout" files.

Reviewer: what is "edit layout" files?

Author: There are several types of layout: _edit_,  _details_ , _close_, and more..

Author: And it fails only in edit files

> * Fixed a bug in the **validate** command failed on "edit layout" files.

Reviewer: ```suggestion
* Fixed a bug in the **validate** command for layout files that had `sortValues` fields.
```
